src/guides/v2.3/security/two-factor-authentication.md

```diff
@@ -71,7 +48,15 @@ After configuring 2FA for your Magento instance, Magento Admin users must instal
 
 ## Troubleshooting
 
-The extension supports command line options to revoke and reset authenticators. Use these commands when you cannot access the Magento Admin.
+The extension supports command line options for disabling, revoking, and resetting authenticators. Use these commands when you cannot access the Magento Admin UI.
+
+### Disable authenticator
```

Comment:
Again, this whole section will not work with the new module. We do not support disabling the module and it will eventually result in a violation of the magento cloud terms of service.

Reply:
I had taken reference to disabling out... looks like some of the old content was put back in. I'll just have to go through it again.